catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -118,7 +119,11 @@ public void onClick(DialogInterface dialog, int which) {
 	public static boolean isNetworkAvailable(Context context) {
 		ConnectivityManager connectivityManager = (ConnectivityManager) context
 				.getSystemService(Context.CONNECTIVITY_SERVICE);
-		return connectivityManager != null && connectivityManager.getActiveNetworkInfo().isConnected();
+		NetworkInfo activeNetworkInfo = null;
+		if (connectivityManager != null) {
+			activeNetworkInfo = connectivityManager.getActiveNetworkInfo();
+		}
+		return activeNetworkInfo != null && activeNetworkInfo.isConnected();
```

Comment:
i did these changes because App crush if there is no active internet connection while taping on register Btn.